### ISSUE TITLE
Server installation mentions FQHN instead of FQDN

### ISSUE THREAD

**Issue by nodeg** [uyuni-project/uyuni#7545]
### Problem description

During the installation of the Uyuni server, @mtravitzky saw an error mentioning `illegal FQHN` and we were both wondering what is meant with that and if it could be a typo or at least not the appropriate term.

@juliogonzalez already found an RFC mentioning an `FQHN`: https://datatracker.ietf.org/doc/html/rfc2109
>    Fully-qualified host name (FQHN) means either the fully-qualified
   domain name (FQDN) of a host (i.e., a completely specified domain
   name ending in a top-level domain such as .com or .uk), or the
   numeric Internet Protocol (IP) address of a host.  The fully
   qualified domain name is preferred; use of numeric IP addresses is
   strongly discouraged.

A simple search in the code revealed 16 files that mention `FQHN`: https://github.com/search?q=repo%3Auyuni-project%2Fuyuni+FQHN&type=code

It needs to be clarified, if the term should be left this way, replaced with `FQDN` or if we should use a different sentence.

### Steps to reproduce

1. Do an Uyuni server installation
2. When setting up the server and specifying an FQDN, use a wrong one
3. Observer the `Illegal FQHN` warning you see in the screenshot below

### Uyuni version

```bash
2023.04
```

### Additional information

![image](https://github.com/uyuni-project/uyuni/assets/12104291/e7fda865-0dbf-4c6c-9e73-7172ff14468c)

**Comment by juliogonzalez** [comment]
@nodeg does YaST support an IP there? And if so... does Uyuni work later?

**Comment by nodeg** [comment]
This needs to be verified by @mtravitzky.

**Comment by juliogonzalez** [comment]
`At least`, but it can be more.

An IPv4 address has three (`1.1.1.1`), and a FQDN/FQHN can have three as well (`uyuni.eu.example.com`) or more.

But on the other hand `IPv6` has none at all.

But I agree, even saying two is ambiguous.

**Comment by agraul** [comment]
What it comes down to is this case: is `uyuni.showtime.` the kind of FQDN Uyuni can work with ?

If not, the message needs an update. It would be enough to include that a trailing dot is not counted towards the two (or phrased differently, we need at least 3 non-empty labels).

**Comment by juliogonzalez** [comment]
Yes, but there's also the question about FQDN vs FQHN... according to what @Bischoff said, the doc clearly says they are not (I never tested it myself, but I don't see a good reason for using an IP to identify the server and emit the SSL certificate, etc, in particular not when remembering IPv6 will be next to impossible).

**Comment by avshiliaev** [comment]
Hey everyone. The conversation is stale for some time already. Do you have any other suggestions?

**Comment by cbosdo** [comment]
Closing as this code is now gone
